### PR TITLE
added NTF_LED_OVERRIDE parameters

### PR DESCRIPTION
**Pull request by tridge**
when this is set the board RGB LED will be controlled by MAVLink
instead of internally. This is useful for cases where the LED patterns
and colours needed are specified by an external authority (such as the
OBC organisers)

### CODE REVIEW

File: libraries/AP_Notify/RGBLed.cpp [hunk 1]
```diff
@@ -323,6 +333,67 @@ void RGBLed::update()
...
+        update_override();
+    }
+}
+
+/*
+  handle LED control, only used when LED_OVERRIDE=1
+*/
+void RGBLed::handle_led_control(mavlink_message_t *msg)
+{
+    printf("Got LED_CONTROL\n");
```
Comment (magicrub) [inline libraries/AP_Notify/RGBLed.cpp]: This print should be commented out

Comment (tridge) [inline libraries/AP_Notify/RGBLed.cpp]: yep, sorry, debugging during development. fixed

**Comment by rmackay9** [comment]
Looks fine to me!

**Comment by tridge** [comment]
applied

**Pull request closed without merging**